src/pykiso/test_coordinator/test_suite.py

```diff
@@ -364,6 +371,69 @@ def __init__(
         # add sorted test case list to test suite
         self.addTests(test_case_list)
 
+        self.failed_suite_setups = set()
+
+    def check_suite_setup_failed(self, test: BasicTest, result: TestResult) -> None:
```

Comment:
I think I need more comments explaining why this works. This is a  complicated part

Reply:
I added a comment which should make it more clearer 
```
Search in the global unittest result object, which save all the results
of the tests performed up to that point, for a BasicTestSuiteSetup tests
which has failed. If the suite setup has failed store the suite id.
```

After i know which suiteID has failed I skipp all tests which belongs to that suide ID.
And thats the trick